app/assets/javascripts/profile.js

```diff
@@ -49,87 +56,125 @@ $(function() {
     // }]
 
     var options = {
-		chart: {
-			renderTo: 'chart',
-            type: 'areaspline'
-        },
-        title: {
-    		text: '',
-    		style: {
-        		display: 'none'
-    		}
-		},
-		subtitle: {
-   			text: '',
-			style: {
-        		display: 'none'
-    		}
-		},
-        legend: {
-            layout: 'horizontal',
-            align: 'right',
-            verticalAlign: 'top',
+  		chart: {
```

Comment:
i was actually trying to clean up unmatched tabs; i'll run beautify on this section and if the auto alignment doesn't look good i can roll back.